v2/jdbc-to-spanner/src/main/java/com/google/cloud/teleport/v2/options/JdbcToSpannerOptions.java

```diff
@@ -0,0 +1,187 @@
+          "The password to be used for the JDBC connection. Can be passed in as a Base64-encoded"
+              + " string encrypted with a Cloud KMS key.")
+  String getPassword();
+
+  void setPassword(String password);
+
+  @TemplateParameter.Text(
+      order = 7,
+      optional = true,
+      groupName = "Source Parameters",
+      description = "The name of a column of numeric type that will be used for partitioning.",
+      helpText =
+          "If this parameter is provided (along with `table`), JdbcIO reads the table in parallel"
+              + " by executing multiple instances of the query on the same table (subquery) using"
+              + " ranges. Currently, only Long partition columns are supported.")
+  String getPartitionColumn();
+
+  void setPartitionColumn(String partitionColumn);
+
+  @TemplateParameter.Text(
+      order = 8,
+      optional = true,
+      groupName = "Source Parameters",
+      description = "Comma-separated names of the tables in the source database.",
+      helpText = "Tables to read from using partitions.")
+  String getTables();
+
+  void setTables(String table);
+
+  @TemplateParameter.Integer(
+      order = 9,
+      optional = true,
+      groupName = "Source",
+      description = "The number of partitions.",
+      helpText =
+          "The number of partitions. This, along with the lower and upper bound, form partitions"
+              + " strides for generated WHERE clause expressions used to split the partition column"
+              + " evenly. When the input is less than 1, the number is set to 1.")
+  Integer getNumPartitions();
+
+  void setNumPartitions(Integer numPartitions);
+
+  @TemplateParameter.Text(
+      order = 10,
+      description = "Cloud Spanner Instance Id.",
+      helpText = "The destination Cloud Spanner instance.")
+  String getInstanceId();
+
+  void setInstanceId(String value);
+
+  @TemplateParameter.Text(
+      order = 11,
+      description = "Cloud Spanner Database Id.",
+      helpText = "The destination Cloud Spanner database.")
+  String getDatabaseId();
+
+  void setDatabaseId(String value);
+
+  @TemplateParameter.ProjectId(
+      order = 12,
+      description = "Cloud Spanner Project Id.",
+      helpText = "This is the name of the Cloud Spanner project.")
+  String getProjectId();
+
+  void setProjectId(String projectId);
+
+  @TemplateParameter.Text(
+      order = 13,
+      optional = true,
+      description = "Cloud Spanner Endpoint to call",
+      helpText = "The Cloud Spanner endpoint to call in the template.",
+      example = "https://batch-spanner.googleapis.com")
+  @Default.String("https://batch-spanner.googleapis.com")
+  String getSpannerHost();
+
+  void setSpannerHost(String value);
+
+  @TemplateParameter.Text(
+      order = 14,
+      optional = true,
+      description = "Source database columns to ignore",
+      helpText =
+          "A comma separated list of (table:column1;column2) to exclude from writing to Spanner",
```

Comment:
Can we use the session file to ignore the columns that are not migrated to Spanner?